docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -65,18 +76,16 @@ The following example shows how to create a gallery-like flexbox layout using th
     toolBar.JustifyContent = FlexContainer.Justification.JustifySpaceBetween;
     ```
 
-5.  Use the toolbar's `Flex` property to make the toolbar and the main content share the height of the main container, so that the toolbar occupies 10 percent of the vertical space and the content occupies the remainder:
+6.  Use `Flex` property of the toolbar to specify occupied area in main container by it. The main container's height is shared between toolbar and content (added in step 7). Since main container displays items vertically and toolbar `Flex` property is set to 0.1, toolbar will occupy 10% of verical and 100% of horizontal available space. Remainded area is intended for content:
```

Comment:
Please check for technical accuracy.

```suggestion
6.  Use `Flex` property of the toolbar to specifes the area to be occupied within the main container. The height main container is shared between toolbar and content as specified in next step. Since the main container displays items vertically and the toolbar `Flex` property is set to 0.1, the toolbar will occupy 10% of vertical and 100% of horizontal available space. The remaining area is intended for the content:

Reply:
Before @pkalota's check, would you please check the "to specifes"? "to specify" seems correct here.

Reply:
Sure, thanks